superset-frontend/src/constants.ts

```diff
@@ -95,6 +95,10 @@ export const URL_PARAMS = {
     name: 'viz_type',
     type: 'string',
   },
+  showDatabaseModal: {
```

Comment:
why is this added?

Reply:
one of the setup guide step is to redirect to database list view and pop open the database modal